shared/actions/chat/inbox.js

```diff
@@ -173,6 +173,12 @@ function* onInboxStale(): SagaGenerator<any, any> {
     yield put(Creators.setInboxUntrustedState('loaded'))
     yield put(Creators.loadedInbox(conversations))
 
+    // Unbox teams so we can get their names
+    yield call(
```

Comment:
we unbox the teams as soon as we can so we get the channel names, else they're unknown to the untrusted server